src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -414,6 +425,7 @@ class ViewAllLogs extends Component {
 
     const resultsFormatter = this.getResultsFormatter();
     const columnMapping = getJobLogsListColumnMapping({ isAllSelected, handleSelectAllCheckbox });
+    const itemToView = sessionStorage.getItem('job-logs.position');
```

Comment:
@OleksandrHladchenko1 I think `storage key` should be the same as `onMarkPosition`  and `resetMarkedPosition` handlers. Also, `itemToView` prop expects `object` so we should parse it.
```suggestion
    const itemToView = JSON.parse(sessionStorage.getItem('@folio/data-import.position'));
```